main/modes/test/keebTest/keebTest.c

```diff
@@ -2,7 +2,7 @@
  * @file keebTest.c
  * @author Jeremy Stintzcum (jeremy.stintzcum@gmail.com)
  * @brief A mode designed to test keyboard variations in rapid succession
- * @version 1.0
+ * @version 1.1
```

Comment:
Only a true professional versions files. Feels like overkill for the Swadge, but I'm cool with it.